macros/src/main/scala/io/udash/macros/PropertyMacros.scala

```diff
@@ -301,10 +275,14 @@ class PropertyMacros(val ctx: blackbox.Context) extends AbstractMacroCommons(ctx
   }
 
   private def subSeqValidation(leafTpe: Type): Unit = {
-    if (!isSeqPropertyTpe(leafTpe)) c.abort(c.enclosingPosition, s"$leafTpe is not a valid subSeq type")
+    val singleParamSeq: Boolean = {
+      val dealiased = leafTpe.map(_.dealias)
+      dealiased.typeConstructor <:< SeqTpe.typeConstructor && dealiased.typeConstructor.typeParams.lengthCompare(1) == 0
```

Comment:
no need for `dealiased.typeParams.lengthCompare(1) == 0` check as well?

Reply:
Nope, type constructor has type params, type has type args. By explicitly getting the type constructor I know what to check.